src/main/java/Event.java

```diff
@@ -0,0 +1,21 @@
+public class Event extends Task{
```

Comment:
There is a minor standard coding violation here. There should be a space between Task and '{'. You may want to check other class file too